scripts/repin.sh

```diff
@@ -10,7 +10,9 @@ set -e
 eval "$(conda shell.bash hook)"
 
 # Create openwpm env with unpinned yaml file
-conda env create --force -q -f environment-unpinned.yaml
+# `PYTHONNOUSERSITE` set so python ignores local user site libraries when building the env
+# See: https://github.com/mozilla/OpenWPM/pull/682#issuecomment-645648939
+PYTHONNOUSERSITE=True conda env create --force -q -f environment-unpinned.yaml
```

Comment:
I think we should also update `./install.sh` to match.

Reply:
Good catch! I added it there as well as during the update command within repin.sh